english/bob-barker/barker.md

```diff
@@ -0,0 +1,15 @@
+The door thuds open and the light coming from within the room in front of you blinds you momentarily.  When your vision clears, standing before you is Bob Barker in all his glory.  
+
+He doesn't look a day past 50.  Like all of the greats he doesn't age in your mind.  
+
+He is flanked by two of the hottest...marshmallows that you have ever seen.  They are literally on fire, like they have been toasting on an open flame and they look delicious.
+
+Bob with a gleam in his eye, looks at you and says, "Come on down you're the next contestant on the .md is right."  You step into the room and walk up to Bob.  Behind him are two large doors.
+
+"Which door is the door for you, the blue door, or the red door?", he asks.
+
+You hesitate just a moment before replying...
+
+[The blue door](../1970s/1970s.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.